### PR TITLE
kill.md: change signal explanation (closes #616)

### PR DESCRIPTION
**Pull request by waldyrious**
I've rebased, improved the descriptions, and fixed linting issues in #616. This had to be done in a separate PR for the reasons explained in #996, but the original commits by @lanzhiwang remain properly attributed.

/cc @igorshubovych, @markusthoemmes, @rubenvereecken, @ostera.

### CODE REVIEW

**Comment by agnivade** [comment]
Love this page. I always forget the signal codes. LGTM !

**Comment by jedahan** [comment]
Really great contribution

File: pages/common/kill.md [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
 `kill {{process_id}}`
 
-- List signal names:
+- List available signal names (to be used without the `SIG` prefix):
 
 `kill -l`
+
+- Terminate a program using the SIGHUP (hang up) signal. Many daemons will reload instead of terminating:
+
+`kill {{1|-HUP}} {{process_id}}`
```
Comment (ostera) [inline pages/common/kill.md]: Shouldn't this be `-1`? And if so, wouldn't it be more concise to have `-{{1|HUP}}` instead?

Comment (waldyrious) [inline pages/common/kill.md]: Good catch! I'll make the changes :)

**Comment by ostera** [comment]
Good stuff guys, keep it up 👍

**Comment by waldyrious** [comment]
@ostera I personally would have liked this one merged rather than squashed, since the commits were logically separated and properly described -- per the description above:

> I've rebased, improved the descriptions, and fixed linting issues in #616. 

 I'll assume it was just an oversight, but if you actually disagree, let me know :)

**Comment by ostera** [comment]
I figured that since the other commits were unattributed it wouldn't be much of an issue – I agree that if we have multiple authors on the same PR, we shouldn't squash to preserve authorship on the individual changes.

**Comment by waldyrious** [comment]
It's not as much about the authorship (although that counts too, to encourage participation). I was more concerned about encouraging (and preserving) the proper separation of commits as atomic changes, with informative descriptions.

**Pull request merged**